recipes/dsdp/bld.bat

```diff
@@ -0,0 +1,11 @@
+
+copy %RECIPE_DIR%\CMakeLists.txt %SRC_DIR%
+copy %RECIPE_DIR%\dsdp.def %SRC_DIR%
+cmake %SRC_DIR% -G "NMake Makefiles" ^
+                    -DCMAKE_BUILD_TYPE=Release ^
+                    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
```

Comment:
Would add `:PATH` right before the `=` to help CMake understand this is a path. Sometimes it has issues with that sort of thing on Windows. So providing CMake some guidance normally helps.